pkgs/applications/editors/jupyter-kernels/clojupyter/default.nix

```diff
@@ -36,8 +36,8 @@ let
     src = fetchFromGitHub {
       owner = "clojupyter";
       repo = "clojupyter";
-      rev = "0.3.2";
-      sha256 = "1wphc7h74qlm9bcv5f95qhq1rq9gmcm5hvjblb01vffx996vr6jz";
+      rev = "0.3.3";
```

Comment:
```suggestion
      rev = version;
```

Reply:
Also a good catch -- sorry for missing. Updated with latest force push.